twindle-web/style.css

```diff
@@ -158,10 +236,21 @@ main .email_subs {
   padding: 2rem 4rem;
 }
 
-main .email_subs .email_input {
```

Comment:
Please can you explain removing .email_input result to what?

Reply:
This is because i need to give different width to input according to view width